src/bioregistry/data/mock_pubmed_data.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
stick this in the tests/ directory - this isn't necessary for people who install the code